src/main/java/org/fundacionjala/sfdc/pageobjects/SalesForceLogIn.java

```diff
@@ -0,0 +1,96 @@
+package org.fundacionjala.sfdc.pageobjects;
+
+import org.fundacionjala.sfdc.commons.DriverManager;
+import org.openqa.selenium.Alert;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.ui.ExpectedConditions;
+import org.openqa.selenium.support.ui.WebDriverWait;
+
+/**
+ * SalesForceLogIn.java
+ * Class to log in Salesforce.
+ */
+public class SalesForceLogIn {
+    private WebDriver webDriver;
+    private String url;
+    private By userName;
+    private By userPassword;
+    private By logInButton;
+    private By cloudIcon;
+    private WebDriverWait wait;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the "wait" field and declare it as a local variable in the relevant methods. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1450)